lambda-functions/jira-bot-filter-task-runner/condition-checker.js

```diff
@@ -0,0 +1,51 @@
+const moment = require('moment');
+const config = require('./config');
+
+lastHumanActivityDate = issue => {
+  const humanHistories = issue.changelog.histories.filter(history => {
+    return config.bots.indexOf(history.author.key) === -1;
+  });
+
+  const humanComments = issue.fields.comment.comments.filter(comment => {
```

Comment:
cross checked what webhook sends as issue. comment.comments is always present and is [] by default